src/settings/toml/manifest.rs

```diff
@@ -13,14 +7,21 @@ use std::path::{Path, PathBuf};
 use config::{Config, File};
 use serde::{Deserialize, Serialize};
 
+use crate::commands::validate_worker_name;
+use crate::settings::toml::deploy_target::{DeployTarget, RouteConfig};
+use crate::settings::toml::environment::Environment;
+use crate::settings::toml::kv_namespace::KvNamespace;
+use crate::settings::toml::site::Site;
+use crate::settings::toml::target_type::TargetType;
+use crate::settings::toml::Target;
 use crate::terminal::emoji;
 use crate::terminal::message;
 
 fn some_string() -> Option<String> {
     Some("".to_string())
 }
 
-#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize)]
```

Comment:
added to allow comparison of Manifests for the benefit of testing constructor methods against one another.